README.md

```diff
@@ -101,6 +101,13 @@ INFO[0000] Function get-python submitted for deployment
 INFO[0000] Check the deployment status executing 'kubeless function ls get-python'
 ```
 
+If you have installed kubeless into some other namespace (which is not called kubeless) or changed the name of the config file from kubeless-config to something else, then you have to export the kubeless namespace and name of kubeless config as environment variables. This can be done as follows:
```

Comment:
I would move this info a little bit before when we explain how to install kubeless:

```
NOTE: Kafka statefulset uses a PVC ...

You can modify the Kubeless manifest if you have some specific requirement but if you install Kubeless into some other namespace (which is not called `kubeless`) or changed the name of the config file from kubeless-config to something else, then you have to export the KUBELESS_NAMESPACE or KUBELESS_CONFIG environment variables in order to reflect the new value. 

You are now ready to create functions.
```